stable/cluster-autoscaler/charts/grafana/templates/k8s-autoscaler.yaml

```diff
@@ -58,7 +58,7 @@ data:
       "editable": true,
       "gnetId": 3831,
       "graphTooltip": 0,
-      "id": null,
+      "id": 36,
```

Comment:
```suggestion
      "id": null,
```

Reply:
I always forget you can suggest commits, I shoulda done that lol